gateway/src/main/java/io/zeebe/gateway/impl/job/ActivateJobsHandler.java

```diff
@@ -75,7 +75,15 @@ private void activateJobs(
                 ResponseMapper.toActivateJobsResponse(key, response);
             final int jobsCount = grpcResponse.getJobsCount();
             if (jobsCount > 0) {
-              responseObserver.onNext(grpcResponse);
+              try {
+                responseObserver.onNext(grpcResponse);
+              } catch (Exception e) {
```

Comment:
Did we ever figure out if other exceptions may be thrown that would be transient and would not indicate that we should stop?

Reply:
Interface documentation doesn't say much. It only says if it throws exception we should call onError() to terminate the stream. Looking at the implementation it can throw also IllegalStateException if the stream is already aborted or completed.

Reply:
Yes. We stop trying other partition.